src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/job/ssh/BuildStartedCommandJob.java

```diff
@@ -59,9 +59,9 @@
      * @param taskListener a listener.
      * @param event        the event.
      * @param stats        the stats.
-     * @see GerritNotifier#buildStarted(AbstractBuild, TaskListener, GerritTriggeredEvent, BuildsStartedStats)
+     * @see GerritNotifier#buildStarted(Run, TaskListener, GerritTriggeredEvent, BuildsStartedStats)
      */
-    public BuildStartedCommandJob(IGerritHudsonTriggerConfig config, AbstractBuild build,
+    public BuildStartedCommandJob(IGerritHudsonTriggerConfig config, Run build,
```

Comment:
This might need to keep its old Constructor signature just in case for backwards compat issues.